<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simulation</class>
 <widget class="QWidget" name="Simulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 0px groove, gray;
    border-color: gray;
	font: 18px, &quot;微软雅黑&quot;;
}

QWidget
{
	background-color: black;
}

QPushButton
{
	font: 18px, &quot;微软雅黑&quot;;
    color: black;
	background-color: rgb(113, 113, 198);
	border:1px solid white;
	border-radius: 5px;
}

QPushButton:pressed
{
	font: 18px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 0, 255);
	border:1px solid white;
	border-radius: 5px;
}

QLineEdit
{
	color: white;
	font: 18px, &quot;微软雅黑&quot;;
}

QCheckBox
{
	font: 15px, &quot;微软雅黑&quot;;
	color: white;
}</string>
  </property>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lndPort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lndByteOffset">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 18px, &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lndBitOffset">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 18px, &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Byte Offset</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bit Offset</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lndValue">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 18px, &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>391</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 18px, &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbType">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>97</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 18px, &quot;微软雅黑&quot;;
background-color: white;
border-color: gray;</string>
   </property>
   <item>
    <property name="text">
     <string>char</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>int</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>uint</string>
    </property>
   </item>
  </widget>
  <widget class="QTextBrowser" name="sendMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>491</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 18px, &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
